spec/requests/api/internal/v1/prosecution_cases_spec.rb

```diff
@@ -40,17 +40,21 @@
                     description: "Searches prosecution cases by prosecution case reference"
 
           parameter name: "include", in: :query, required: false, type: :string,
-                    schema: {},
-                    description: 'Return defendant, offence and hearing data through a has_many relationship </br>
-                                  eg include=defendants,defendants.offences,defendants.defence_organisation,hearings.providers,hearing_summaries'
+                    schema: {
+                      "$ref": "prosecution_case.json#/definitions/example_included_query_parameters",
+                    },
+                    description: 'Include top-level and nested associations for a prosecution case. All top-level and nested
+                                  associations available for inclusion are listed under the relationships key of the response body.
+                                  e.g. to include hearing_summaries, hearings, defendants and their offences: </br>
+                                  eg include=hearing_summaries,hearings,defendants,defendants.offences'
```

Comment:
Delete this 'eg'

This description on the UI seems to come from the swagger.yaml, not here anyway but still.